terraform-apstra-examples/apstra_device_config/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_providers {
+    apstra = {
+      source  = "Juniper/apstra"
```

Comment:
I recommend that you set the provider version to minimum that has the config getter thingy